docs-chef-io/content/inspec/reporters.md

```diff
@@ -15,7 +15,7 @@ A `reporter` is a facility for formatting and delivering the results of a Chef I
 
 Chef InSpec allows you to output your test results to one or more reporters.
 
-Configure the reporter(s) using either the `--reporter` option or as part of the general configuration file using the `--config` (or `--json-config`, prior to v3.6) option. While you can configure multiple reporters to write to different files, only one reporter can output to the screen(stdout).
+Configure the reporter(s) using the `--reporter` option or as part of the general configuration file using the `--config` (or `--json-config`, prior to v3.6) option. Or it could also be configured by using both the options together. While you can configure multiple reporters to write to different files, only one reporter can output to the screen(stdout).
```

Comment:
How about "Both the --reporter and --config options may be used, in which case the options are merged."
As is, it reads like you are suggesting both --config and --json-config may be used.